jax/_src/scipy/signal.py

```diff
@@ -164,7 +165,14 @@ def _convolve_nd(in1: Array, in2: Array, mode: str, *, precision: PrecisionLike)
   if not (no_swap or swap):
     raise ValueError("One input must be smaller than the other in every dimension.")
 
+  shape1 = in1.shape
   shape_o = in2.shape
+  if in1.ndim > 1:
+    M = shape1[0] if shape1[0] >= shape_o[0] else shape_o[0]
+    N = shape1[1] if shape1[1] >= shape_o[1] else shape_o[1]
+    if boundary == 'wrap' and mode != 'valid':
+      in1 = jnp.pad(in1, (M, N), mode='wrap')
```

Comment:
The `convolve` function currently lacks support for the `boundary` argument with 1D arrays. Because of this, the `boundary='wrap'` mode is ignored in this case. For multi-dimensional inputs (3D or higher), can we raise a `NotImplementedError`.

Reply:
Sounds good. We should also raise `NotImplementedError` in `_nd_convolve` for 1D inputs if we're not implementing that.